package_manifest.yaml

```diff
@@ -923,88 +923,11 @@ repoclosures:
           - el8-puppet-7
           - "el8-foreman-{{ foreman_version }}"
           - "el8-foreman-plugins-{{ foreman_version }}"
-          - "el8-katello-candlepin-{{ katello_version }}"
-    foreman-client-repoclosure-el9:
-      repoclosure_target_repos:
-        el9:
-          - "el9-foreman-client-{{ foreman_version }}"
-      repoclosure_target_dist: el9
-      repoclosure_lookaside_repos:
-        el9:
-          - el9-baseos
-          - el9-appstream
-          - el9-puppet-7
-    foreman-client-repoclosure-el8:
-      repoclosure_target_repos:
-        el8:
-          - "el8-foreman-client-{{ foreman_version }}"
-      repoclosure_target_dist: el8
-      repoclosure_lookaside_repos:
-        el8:
-          - el8-baseos
-          - el8-appstream
-          - el8-powertools
-          - el8-epel
-          - el8-extras
-          - el8-puppet-7
-    foreman-client-repoclosure-el7:
-      repoclosure_target_repos:
-        el7:
-          - "el7-foreman-client-{{ foreman_version }}"
-      repoclosure_target_dist: el7
-      repoclosure_lookaside_repos:
-        el7:
-          - el7-base
-          - el7-updates
-          - el7-epel
-          - el7-extras
-          - el7-puppet-7
-    foreman-staging-repoclosure-el8:
-      repoclosure_target_repos:
-        el8:
-          - "el8-foreman-{{ foreman_version }}-staging"
-      repoclosure_target_dist: el8
-      repoclosure_lookaside_repos:
-        el8:
-          - el8-baseos
-          - el8-appstream
-          - el8-powertools
-          - el8-extras
-          - el8-puppet-7
-          - "el8-candlepin-{{ candlepin_version }}-staging"
-    plugins-staging-repoclosure-el8:
-      repoclosure_target_repos:
-        el8:
-          - "el8-foreman-plugins-{{ foreman_version }}-staging"
-      repoclosure_target_dist: el8
-      repoclosure_lookaside_repos:
-        el8:
-          - el8-baseos
-          - el8-appstream
-          - el8-powertools
-          - el8-configmanagement-salt
-          - el8-extras
-          - el8-puppet-7
-          - "el8-foreman-{{ foreman_version }}-staging"
-    katello-staging-repoclosure-el8:
-      repoclosure_target_repos:
-        el8:
-          - "el8-katello-{{ katello_version }}-staging"
-      repoclosure_target_dist: el8
-      repoclosure_lookaside_repos:
-        el8:
-          - el8-baseos
-          - el8-appstream
-          - el8-powertools
-          - el8-extras
-          - el8-puppet-7
-          - "el8-foreman-{{ foreman_version }}-staging"
-          - "el8-foreman-plugins-{{ foreman_version }}-staging"
-          - "el8-candlepin-{{ candlepin_version }}-staging"
+          - "el8-candlepin-{{ candlepin_version }}"
     foreman-client-staging-repoclosure-el9:
       repoclosure_target_repos:
         el9:
-          - "el9-foreman-client-{{ foreman_version }}-staging"
+          - "el9-foreman-client-{{ foreman_version }}"
```

Comment:
I must admit I liked the `-staging` suffix, as it indicated correctly that we're not looking at the prod repos, but staging.

Not going to block on this tho :)

Reply:
I liked it too... I just assumed that others would want a clean revert back to the way things were with koji gone. I'd be happy to keep the staging nomenclature.

Reply:
:+1: on keeping staging explicitly named.